### PR TITLE
Created new contact creation form UI

### LINKED ISSUE #5: Create a new contact UI

### PR DESCRIPTION
**Pull request by allenjiji**
Fixes #5 
Also considered the review corrections for PR #11 .
image
image

Requesting a review.
@yedhink

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Form";
+
+const NewContactPane = ({ showPane, setShowPane }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx:7]: Go through the Slack thread. The component name and the file name have disparity.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import { Pane, Button, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import {
+  ROLES,
+  CONTACT_FORM_VALIDATION_SCHEMA,
+  CONTACT_FORM_INITIAL_VALUES,
+} from "../constants";
+
+const ContactCreationForm = ({ onClose, isEdit }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:14]: You component naming is wrong. Read: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+              label="Email"
+              name="email"
+              placeholder="Enter your email"
+              type="email"
+            />
+            <Select
+              required
+              className="w-full flex-grow-0"
+              label="Role"
+              name="userRole"
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:61]: Why do you have to prefix with the `user` keyword here?

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+              size="large"
+              style="primary"
+              type="submit"
+              onClick={() => setSubmitted(true)}
+            />
+            <Button
+              label="Cancel"
+              size="large"
+              style="text"
+              onClick={onClose}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:82]: Go through the Slack thread.

File: app/javascript/src/components/Dashboard/Contacts/constants.jsx [hunk 5]
```diff
@@ -40,12 +41,10 @@ const CONTACTS_TABLE_COLUMNS = [
...
     render: () => (
-      <div>
-        <Dropdown buttonStyle="text" icon={MenuHorizontal} strategy="fixed">
-          <li> Edit </li>
-          <li> Delete </li>
-        </Dropdown>
-      </div>
+      <Dropdown buttonStyle="text" icon={MenuHorizontal} strategy="fixed">
+        <li> Edit </li>
+        <li> Delete </li>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.jsx:46]: Why is there a whitespace before and after the text, `Edit` and `Delete`?

**Review by yedhink — COMMENTED** [review]
@allenjiji

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx (hunk 4 above)
Comment (allenjiji) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:82]: Sorry, I missed the type="reset" prop for the cancel button.

**Pull request merged**